interfaces/builtin/opengl.go

```diff
@@ -103,6 +103,8 @@ const openglConnectedPlugAppArmor = `
 # nvidia
 /etc/vdpau_wrapper.cfg r,
 @{PROC}/driver/nvidia/params r,
+@{PROC}/driver/nvidia/gpus/[0-9]*:[0-9]*:[0-9]*.[0-9]*/information r,
```

Comment:
is the 'bus location' known to be only pci-like, or would it make sense to use more generic pattern here?

```suggestion
@{PROC}/driver/nvidia/gpus/*/information r,
```

Reply:
Honestly I'm not sure.

I was going for the safest default, but I would be in favour of switching to a more generic pattern too for simplicity/compatibility if there is no objection.

Reply:
I've updated to the wildcard.  I can revert if anyone objects.